src/instantx.cpp

```diff
@@ -983,22 +988,9 @@ bool CTxLockRequest::IsValid() const
         return false;
     }
 
-    CAmount nValueOut = tx->GetValueOut();
-
-    if(nValueIn - nValueOut < GetMinFee()) {
-        LogPrint("instantsend", "CTxLockRequest::IsValid -- did not include enough fees in transaction: fees=%d, tx=%s", nValueOut - nValueIn, ToString());
-        return false;
-    }
-
     return true;
 }
 
-CAmount CTxLockRequest::GetMinFee() const
-{
-    CAmount nMinFee = MIN_FEE;
-    return std::max(nMinFee, CAmount(tx->vin.size() * nMinFee));
-}
-
```

Comment:
Same, should modify this to handle simple txes correctly instead of dropping this.